freqtrade/plugins/pairlist/IPairList.py

```diff
@@ -39,6 +39,12 @@ class __OptionPairlistParameter(__PairlistParameterBase):
     options: List[str]
 
 
+class __ListPairListParamenter(__PairlistParameterBase):
+    type: Literal["list"]
+    default: Union[List[str], None]
+    options: List[str]
```

Comment:
```suggestion
    default: Union[List[str], None]
```

i don't think options makes sense here - it's a "random" list ... 
or was there a deeper use you thought about with this? 

**edit** this is also what's causing the mypy issue